salt/utils/proxy.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Utils for proxy.
+"""
+
+# Import Python libs
```

Comment:
Let's also drop the import comment headers.
Our automated formatting sometimes moves stuff around and those headers stop making sense.
Also, isort, these days maintains a constant order that everyone will, eventually, get accustomed to.

Reply:
Done.